firmware/sys/storage/include/storage.h

```diff
@@ -28,11 +28,18 @@
 #define STORAGE_H
 
 #include <stdio.h>
+#include "mtd_flashpage.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define    LAST_PAGE 1
+
+/**
+ * @brief this is the number of pages that we can write
+ */
+#define LAST_AVAILABLE_PAGE (FLASHPAGE_NUMOF - last_page)
```

Comment:
Ok looks better the macro in uppercaps, but the same change, should be here with last_page